File: C/FizzBuzz/fizzBuzz.c
#include <stdio.h>

void fizzBuzz(int num, char* output) {
    if (num == 0) {
        sprintf(output, "%d", num);
        return;
    }
    if (num % 3 == 0 && num % 5 == 0)
        sprintf(output, "FizzBuzz");
    else if (num % 3 == 0)
        sprintf(output, "Fizz");
    else if (num % 5 == 0)
        sprintf(output, "Buzz");
    else
        sprintf(output, "%d", num);
}

File: Java/CyberDojo/fizzBuzz/src/FizzBuzz.java
public class FizzBuzz {
    public static void main(String... args) {
        for(int number = 1; number <= 100; number++)
            System.out.println(fizzBuzz(number));
    }
    public static String fizzBuzz(int number) {
        if (number < 0) throw new IllegalArgumentException("Number must be positive");
        if (number == 0) return "0";
        StringBuilder result  = new StringBuilder();
        if (number % 3 == 0) result.append("Fizz");
        if (number % 5 == 0) result.append("Buzz");
        if (result.isEmpty()) result.append(number);
        return result.toString();

    }
}
